website/addons/s3/tests/test_model.py

```diff
@@ -6,179 +6,90 @@
 from tests.factories import UserFactory, ProjectFactory
 
 from framework.auth import Auth
-from website.addons.s3.model import AddonS3NodeSettings, AddonS3UserSettings
+from website.addons.base.testing import models
+from website.addons.s3.model import S3NodeSettings, S3UserSettings
```

Comment:
`S3UserSettings` and `UserFactory` are not being used and can be removed. Same with `from boto.s3.connection import *` (not shown in the diff)